makefile

```
@@ -1,21 +1,83 @@
 .DEFAULT_GOAL=compile-pypi
```

need changing to match new target - 
```suggestion
.DEFAULT_GOAL=compile_and_test-pypi-arm64
```